packages/imagemagick7.rb

```diff
@@ -67,6 +62,8 @@ def self.build
   end
 
   def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/include"
+    FileUtils.ln_s "#{CREW_PREFIX}/include/locale.h", "#{CREW_PREFIX}/include/xlocale.h", force: true
```

Comment:
This line should be:
```
FileUtils.ln_sf "#{CREW_PREFIX}/include/locale.h", "#{CREW_DEST_PREFIX}/include/xlocale.h"
```
That said, what happens if you patch the configure script to look for locale.h instead of xlocale.h?  Would that eliminate the need for the symlink?

Reply:
I don't know. I was thinking to patch the source files themselves... I'll try that this evening. The qtfm maintainer doesn't seem to be very active, so patching would be a better solution rather than submitting a PR

Reply:
@uberhacker I managed to avoid the xlocale.h symlink. This needs to patch imagemagick7 source files but also recompile perl.rb, which had an old dependency to xlocale.h in its binaries. This led to the need to add a symlink named libnsl.so. This lib is produced by glibc. I thought it was less impacting to insert it as close as possible to the actual need instead of insert it close to its origin.